spring-rest2ts-angular2json/spring-rest2ts-angular2json-impl/src/main/java/com/blueveery/springrest2ts/converters/ModelClassesToTsAngular2JsonApiConverter.java

```diff
@@ -148,7 +95,7 @@ private TSDecorator createJsonApiModelConfigDecorator(Class javaClass, TSClass t
         JsonApiModelConfig jsonApiModelConfig = (JsonApiModelConfig) javaClass.getAnnotation(JsonApiModelConfig.class);
         if (jsonApiModelConfig != null) {
             typeName = jsonApiModelConfig.type();
-        }else{
+        } else {
```

Comment:
For better readability, wouldn't it better to shorten this logic to:
``final String typeName = (jsonApiModelConfig != null) ? jsonApiModelConfig.type() : tsClass.getName().toLowerCase() + "s";``